.github/workflows/ci-main.yml

```diff
@@ -36,6 +36,10 @@ jobs:
           npm run build || vite build || make
       - name: Firefox build
         run: make firefox
+      - name: zip source code
+        run: |
+          zip blue-blocker-source-$(make version).zip -r ./* -x ".git/*" "build/*" \
```

Comment:
`actionlint` says:
```
.github/workflows/ci-main.yml:40:9: shellcheck reported issue in this script: SC2046:warning:1:25: Quote this to prevent word splitting [shellcheck]
   |
40 |         run: |
   |         ^~~~
```
...so I guess it's complaining about the `$(make version)` part? That looks kinda sketchy to begin with... is that really the best way to get the version number?

Reply:
Also, `yamllint` says:
```
  41:81     error    line too long (84 > 80 characters)  (line-length)
```
...so can you wrap it a little more tightly while you're at it?

Reply:
Fixed both of these